resources/startPage.html.template

```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'none'; img-src {{ cspSource }} file:; script-src 'nonce-{{ nonce }}'; font-src {{ codiconsFontUri }}; style-src {{ cspSource }} {{ codiconsStyleUri }} 'nonce-{{ nonce }}';" />
+    <link href="{{ codiconsStyleUri }}" rel="stylesheet" />
+    <title>Docker - Get Started</title>
+
+    <style nonce="{{ nonce }}">
+        #titleBar {
+            align-items: center;
+        }
+
+        .icon .codicon {
+            font-size: 70px;
+            padding: 5px;
+        }
+
+        #dockerIcon {
+            width: 70px;
+            padding: 5px;
+        }
+
+        #dockerIcon path {
+            fill: var(--vscode-editor-foreground);
+        }
+
+        a.icon:hover {
+            color: var(--vscode-editor-foreground);
+        }
+
+        div .container {
+            display: flex;
+        }
+
+        div .row {

```

Comment:
A grid might make more sense for the list of feature icons/descriptions (just in case there's any width differences between the different icons, the columns would always line up):

```
.container {
  display: grid;
  grid-gap: 10px;
  grid: auto-flow / auto 1fr;
}
```

Reply:
Each direct child of the container element would alternate between the first column and the second column position on increasing rows.

Reply:
I was able to do it with Flexbox by just setting the width on the Docker logo the same as the icons get.